js/widgets/panel.js

```diff
@@ -30,29 +30,27 @@
 	}
 } )( function( $ ) {
 
+var classesMap = {
+	pageContainer: "ui-panel-page-container",
+
+	// Used for wrapper and fixed toolbars position, display and open classes.
+	pageContentPrefix: "ui-panel-page-content"
+};
+
 return $.widget( "mobile.panel", {
 	version: "@VERSION",
 
 	options: {
 		classes: {
-			panel: "ui-panel",
-			panelOpen: "ui-panel-open",
-			panelClosed: "ui-panel-closed",
-			panelFixed: "ui-panel-fixed",
-			panelInner: "ui-panel-inner",
-			modal: "ui-panel-dismiss",
-			modalOpen: "ui-panel-dismiss-open",
-			pageContainer: "ui-panel-page-container",
-			pageWrapper: "ui-panel-wrapper",
-			pageFixedToolbar: "ui-panel-fixed-toolbar",
-			pageContentPrefix: "ui-panel-page-content", /* Used for wrapper and fixed toolbars position, display and open classes. */
-			animate: "ui-panel-animate"
+			"ui-panel": "ui-panel-closed"
```

Comment:
@arschmitz can correct me if I'm wrong, but I don't believe ```ui-panel-closed``` is supposed to be a value, but rather a key. The value of a class key is supposed to be a set of style-related classes, IINM.